nri/index.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8">
+    <title>NRIの強み　ビジネスモデルの特徴 | 企業情報 | 野村総合研究所(NRI)
+    </title>
+    <meta name="descripton" content="野村総合研究所（NRI）の公式ホームページです。NRIからの提言や調査・レポート、商品サービス、ITソリューション事例、IR情報、採用情報、CSR情報などを掲載しています">
+    <link rel="icon" type="images/png" href="images/favicon.ico">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Cabin">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header>
+      <h3><a href="#"><img class="header-logo" src="images/apple-touch-icon.png" alt="NRI公式ホームページトップ"></a>
```

Comment:
別のところでも指摘をいれていますが、見出しタグは

- h1の中にh2
- h2の中にh3
- ...

という包括関係になるよう記述します。

なので、いきなりh3が出てくることはありません。

とはいえ、h1は文章のタイトル（メインコンテンツのタイトル）になるので、この部分をh1にするのは誤りです。
今回のh1は「NRIのデジタルビジネス戦略」が適切ですね。（この部分はできている）。

```suggestion
      <span><a href="#"><img class="header-logo" src="images/apple-touch-icon.png" alt="NRI公式ホームページトップ"></a></span>
```

とか

```suggestion
      <p><a href="#"><img class="header-logo" src="images/apple-touch-icon.png" alt="NRI公式ホームページトップ"></a></p>
```

とかが適切かなと思います。（あえて見出しにするほどではないので）